src/components/MarketOpeningTimeCounter.jsx

```diff
@@ -1,35 +1,26 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { displayMilliseconds } from '../utils/index';
 import ServerTime from '../utils/ServerTime';
 
-export class MarketOpeningTimeCounter extends React.Component {
-    constructor(props) {
-        super(props);
-        this.serverTime = ServerTime.getInstance();
+export const MarketOpeningTimeCounter = ({ symbolOpenTime }) => {
+    const [time, setTime] = useState(0);
```

Comment:
can we use `React.useState` to be consistent?

Reply:
@balakrishna-binary just fixed it